integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -100,19 +100,22 @@ spec:
             {{- end }}
           {{- end }}
           command: ["/entrypoint.sh"]
-          {{- if .Values.fuse.args }}
           args:
-{{ toYaml .Values.fuse.args | trim | indent 12 }}
-          {{- end }}
+            - fuse
+            {{- if .Values.fuse.mountOptions }}
+            - --fuse-opts={{ .Values.fuse.mountOptions }}
+            {{- end }}
+            {{- if .Values.fuse.mountPath }}
+            - {{ .Values.fuse.mountPath }}
+            {{- end }}
+            {{- if .Values.fuse.alluxioPath }}
```

Comment:
lol what if no ` .Values.fuse.mountPath` but has `.Values.fuse.alluxioPath`

Reply:
The `alluxio-fuse` script will regard `alluxioPath` as the mount point and issue a warning there.

The reason I'm not checking here is that, using CLI to mount has the same problem: if user only provides one argument, in `alluxio-fuse` we use it as the mount point, not alluxioPath. So I think addressing in`alluxio-fuse` is more suitable because it's not a k8s-specific problem